Resources/doc/reference/filter_field_definition.rst

```diff
@@ -179,3 +179,31 @@ implement this functionality.
             return true;
         }
     }
+
+Filtering case insensitive strings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Normally the query builder behind the filtering would compare field values case sensitive. There can be reasons to
+filter case insensitive. To do so you can use The ``compare_case_insensitive`` option to force a case insensitive
```

Comment:
I was on my phone when I wrote this and omitted to put "the" in lowercase, I thought you would naturally fix it :P